Parkduksung/ElevenStreetSearchExample/app/src/main/java/com/exam/elevenstreet/view/product/adapter/ProductAdapter.kt

```diff
@@ -0,0 +1,45 @@
+package com.exam.elevenstreet.view.product.adapter
+
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.exam.elevenstreet.R
+import com.example.elevenstreet.ProductResponse
+import kotlinx.android.synthetic.main.list_layout.view.*
+
+
+class ProductAdapter(val productList: ArrayList<ProductResponse>) : RecyclerView.Adapter<ProductAdapter.ViewHolder>() {
+    override fun onBindViewHolder(holder: ViewHolder, position: Int) {
+        val product: ProductResponse = productList[position]
+
+        holder.productName.text = product.productName
+        holder.productPrice.text = product.productPrice
+        holder.productCode.text = product.productCode
+        holder.productImage.text = product.productImage
+
+
+    }
+
+    override fun onCreateViewHolder(holder: ViewGroup, position: Int): ViewHolder {
+        val view = LayoutInflater.from(holder.context).inflate(R.layout.list_layout, holder, false)
+        return ViewHolder(view)
+    }
+
+
+    override fun getItemCount(): Int =productList.size
```

Comment:
```suggestion
    override fun getItemCount(): Int =
         productList.size
```
함수에 = 을 사용한 경우, 해당 convention 을 따르는 것이 좋을 것 같습니다.
https://kotlinlang.org/docs/reference/coding-conventions.html#expression-body-formatting

Reply:
`77b5fd8`  변경은 했다만 "nitializer를 사용하는 속성의 경우, initializer가 긴 경우 등호 뒤에 줄 바꿈을 추가하고 initializer를 4개의 공백으로 삽입하십시오." 라는 구문해석인데 물론 통일하는건 좋지만 긴경우는 안해도되지안나염?ㅋㅋㅋㅋㅋ

Reply:
![image](https://user-images.githubusercontent.com/48197016/65296668-19772600-dba0-11e9-81f9-b09231a710aa.png)
이걸 말씀드린거고 보신건 함수가 아닌 변수에대한 convention입니다.

Reply:
아하 ㅋㅋㅋㅋㅋ 그렇구만 ㅋㅋㅋ 역시는 역시군

Reply:
@innovator777 이러한이유로 저도 한줄로 하다가 한줄 내려서 사용하였습니다.